### DISCUSSION TITLE
multiple connections to cast voice

### DISCUSSION THREAD

**Discussion by huhas12** [AzuraCast/AzuraCast#3866] [Q&A]
Is it possible for two mics being in different places (IPs) to cast voice at the same time?

**Comment by uninventiveheart** [comment]
That's against the design of both Icecast and Shoutcast.  Both are designed to only permit one source IP to access the server for streaming.  It's a common question that crops up that can be addressed client-side: try using a streaming app like Mixxx, VB-Cable or Virtual Audio Cable, and a voice chat app like Discord or Skype.  Then the second IP is simply an Auxiliary or Mic source to you.

Latency is an issue, but that would be the case regardless: radio always has a delay from the streamer/DJ to the listener.

**Reply by ghost** [reply]
I'm using FarPlay.io for multiple DJs in different locations with VB-Cable
